tomls/omnibus-arbitrum-sepolia/spot/sol.toml

```
@@ -0,0 +1,70 @@
+[var.sol_spot_settings]
```

```suggestion
[var.SOL_spot_settings]
```
it would be good to keep symbol consistent everywhere

---

alright sure thing, god help me navigate through all this 😅